fbpush.sh

```diff
@@ -15,6 +15,43 @@ command -v hub >/dev/null 2>&1 || {
     exit 1
 }
 
+#  check where hub config is if it exists
+#  either if XDG_CONFIG_HOME or XDG_CONFIG_DIRS or standard directory
+
+STANDARD_HUB_PATH="/home/$(whoami)/.config/hub"
+HUB_PATH="/home/$(whoami)/.config/hub"
+
+if [ -v XDG_CONFIG_HOME ]; then
+    echo "Look in XDG_CONFIG_HOME"
+    HUB_PATH="$XDG_CONFIG_HOME/hub"
+    if [ ! -e $XDG_CONFIG_HOME ]; then
+        echo "Hub directory $XDG_CONFIG_HOME does not exist"
+        exit 1
+    fi
+elif [ -v XDG_CONFIG_DIRS ]; then
+    echo "Look in XDG_CONFIG_DIRS"
+    for path in ${XDG_CONFIG_DIRS//:/ }; do
+        if [ -e $path ]; then
+            HUB_PATH="$path/hub"
+            echo "Use directory $path"
+            break
+        fi
+    done
+fi
+
+if [ ! -e "$HUB_PATH" ]; then
+    echo "Hub config file $HUB_PATH does not exist"
+fi
+
+echo "Check hub standard path : $STANDARD_HUB_PATH"
+
+if [ ! -e "$STANDARD_HUB_PATH" ]; then
+    echo "Hub config file $STANDARD_HUB_PATH does not exist"
+    exit 1
+fi
+
```

Comment:
Warum diese Trennung zwischen standard und "normal". Warum wird beides gecheckt?

Reply:
das war falls es in dem einen verzeichnis keine config datei gibt, hab ich weg gemacht